src/shogun/features/streaming/StreamingDenseFeatures.cpp

```diff
@@ -49,7 +49,6 @@ template<class T> CStreamingDenseFeatures<T>::CStreamingDenseFeatures(
 	init(file, is_labelled, size);
 	set_read_functions();
 	parser.set_free_vector_after_release(false);
-	parser.set_free_vectors_on_destruct(false);
```

Comment:
so I guess you added these lines for a reason... Not sure why you would remove them, as the only failing unit test related to this is the one you added below, see comments there